### ISSUE TITLE
Modules and FFI don't go well together.

### ISSUE THREAD

**Issue by elordin** [links-lang/links#292]
Minimal example:
```
# a.links
open B
42
```
```
# b.links
alien javascript "/my/js/file.js" {
  foo: () ~> ();
}
```

```
links -m a.links
```
yields
```
*** Error: "Assert_failure refineBindings.ml:265:23"
```

**Comment by SimonJF** [comment]
Why am I not surprised :)

I think the module system is beginning to be more trouble than its worth, and is due a redesign.

On 13/10/17 17:44, Thomas Weber wrote:

Minimal example:

# a.links
open B
42


# b.links
alien javascript "/my/js/file.js" {
  foo: () ~> ();
}


links -m a.links


yields

*** Error: "Assert_failure refineBindings.ml:265:23"

**Comment by SimonJF** [comment]
Funnily enough, this is fixed as part of #463 (although due to Proper Modules coming along, I doubt it's relevant).
